tests/e2e/kubectl.py

```diff
@@ -125,7 +125,7 @@ def create_ns(ns):
     launch(f"get ns {ns}", ns=None)
 
 
-def delete_ns(ns, ok_to_fail=False, timeout=600):
+def delete_ns(ns, ok_to_fail=False, timeout=6000):
```

Comment:
Are you kidding?

Reply:
Sometimes 60 seconds timeout is hit when running tests in docker

Reply:
However, I will add if-else to set lower timeout for native execution
Thanks for noting

Reply:
I can understand 600, but 6000?!!!

Reply:
AFAIR, 6000 here means 600 seconds. However, I may be wrong.

Reply:
It is seconds